Tests/SwiftDocCTests/Infrastructure/DocumentationContextTests.swift

```diff
@@ -553,8 +553,24 @@ class DocumentationContextTests: XCTestCase {
             .flatMap { $0.variants.map { $0.value.lastPathComponent } }
             .sorted()
         
-        XCTAssertEqual(["figure1.jpg", "figure1.png", "figure1~dark.png", "intro.png", "introposter.png", "introposter2.png", "something@2x.png", "step.png", "titled2up.png", "titled2upCapital.PNG", "with spaces.png", "with spaces@2x.png"], imagesRegistered.sorted())
-    }
+        XCTAssertEqual(
+          [
+            "figure1.jpg",
```

Comment:
This one is still using a two-space indent. I'd expect 4 spaces here for consistency